### PR TITLE
lights off

### PR DESCRIPTION
**Pull request by ArnoldSmith86**
I had this idea yesterday that this might look nice on phones with AMOLED screens when using fullscreen.

This PR adds a new toolbar button after the sound button that toggles between the normal background and toolbar and black versions with dimmed images.

Please make suggestions about where to put this. The players overlay is already called "Players & Settings" - maybe this is the point where we move sound and lights there?

### CODE REVIEW

**Comment by ArnoldSmith86** [comment]
PR-SERVER-BOT: You can play around with it here: https://test.virtualtabletop.io/PR-1154/pr-test (or any other room on that server)

**Comment by 96LawDawg** [comment]
I tested it on Windows in Edge, Chrome, and Firefox.  I tested it on iPhone and iPad with Safari in portrait and landscape.  It worked and looked good.  BUT, I looked at the commit and you changed some audio code.  It looks like you were just trying to clean it up and simplify it.  But you broke it.  Audio no longer works in this PR.

As for where to put your light's out feature, for now, I say leave it where it is.  The current options fit on my iPhone in portrait mode, just barely.  If we add anything else we'll have to make some decisions about organization and menu trees.  But for now we don't so I say leave it alone.

I would approve this as is if it didn't brake audio.

File: client/js/main.js [hunk 1]
```diff
@@ -247,29 +247,34 @@ onLoad(function() {
 
   on('#muteButton', 'click', function(){
     if(muted) {
-      document.getElementById('volume').value = unmuteVol;
-      document.getElementById('muteButton').classList.remove('muted');
-      var allAudios = document.querySelectorAll('audio');
-      allAudios.forEach(function(audio){
+      $('#volume').value = unmuteVol;
+      $('#muteButton').classList.remove('muted');
+      $a('audio').forEach(function(audio){
```
Comment (robartsd) [inline client/js/main.js:252]: I'm not familiar with the use of `$a`

**Comment by ArnoldSmith86** [comment]
> Audio no longer works in this PR.

Sorry. GitHub was refusing to handle this PR correctly so I wanted another commit (which did fix it). I will take another look and either fix or revert it.

**Comment by ArnoldSmith86** [comment]
> I would approve this as is if it didn't break audio.

Does it work in other recent PRs? Because it looks like it can't find the audio assets.

**Comment by robartsd** [comment]
> > I would approve this as is if it didn't break audio.
> 
> Does it work in other recent PRs? Because it looks like it can't find the audio assets.

It doesn't look like a url prefix issue unless audio assets are handled under a different path.
https://github.com/ArnoldSmith86/virtualtabletop/blob/main/client/js/widgets/widget.js#L1567

**Comment by ArnoldSmith86** [comment]
Please test if audio works in #1155.

**Comment by 96LawDawg** [comment]
I just tested in #1141  right as you were writing that.  It is the switch to test.virtualtabletop.io that is causing the problem.  If you press the play sound button in or in this one, the sound works.  But the assets that are the sounds in the AUDIO tutorial are not being loaded into the test server.

**Review by 96LawDawg — APPROVED** [review]
Approved.  The issues I had with AUDIO were not related to this and will be fixed by #1155.  You may want more input on whether the feature should be part of the players and settings menu or not, but as I said before, until we add one more menu option, then this still fits and can be left where it is.

**Review by RaphaelAlvez — APPROVED** [review]
tested and I really like the results

**Comment by ArnoldSmith86** [comment]
> Approved. The issues I had with AUDIO were not related to this and will be fixed by #1155.

Please test again now that #1155 is integrated.

**Review by 96LawDawg — APPROVED** [review]
Audio still works and the lights off feature works as attended.  Recommend merging.

File: client/js/main.js (hunk 1 above)
Comment (ArnoldSmith86) [reply client/js/main.js:252]: `$` gives you the first matching element; `$a` gives you a collection of all matching elements (basically just a shorthand for `document.querySelectorAll`.

**Pull request merged**